python/paddle/v2/fluid/framework.py

```diff
@@ -1123,3 +1122,59 @@ def program_guard(main_program, startup_program=None):
     switch_main_program(main_program)
     if startup_program is not None:
         switch_startup_program(startup_program)
+
+
+def debug_block_image(block, highlights=[], path="./temp.dot"):
```

Comment:
I think it's better to move this function to an independent file

Reply:
fluid.debuger ?

Reply:
cool, I think that is ok.